### PR TITLE
Get-EntraAttributeSet

### PR DESCRIPTION
**Pull request by v-akarke**
Added customization and Unit test cases

### CODE REVIEW

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraAttributeSet.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+## DESCRIPTION
+
+Gets a list of Microsoft Entra ID attribute sets.
+
+## EXAMPLES
+
+### Example 1: Get an all attribute sets.
+```powershell
+Get-EntraAttributeSet
```
Comment (KenitoInc) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraAttributeSet.md:47]: Add `Connect-Entra` command with the correct scopes

Comment (KenitoInc) [reply module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraAttributeSet.md:47]: Scope is `CustomSecAttributeDefinition.Read.All`

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraAttributeSet.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+```Output
+Id                    Description                           MaxAttributesPerSet
+--                    -----------                           -------------------
+engineering           Attributes for cloud engineering team 25
+Test2                 Attributes for test2  team            25
+demo                  NEW Demo Description                  25
+demo2                 TEST THE DEMO
+DEMO5
+DEMO6                 Description to test the demo6
+DEMO7                 Description to test the demo6         24
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraAttributeSet.md:71]:
```suggestion
Engineering           Attributes for cloud engineering team 25
HR                 Attributes for HR team            15
Hackathon                  Hackathon attribute set                  20
```

Comment (SteveMutungi254) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraAttributeSet.md:71]:
```suggestion
Id          Description                            MaxAttributesPerSet
--          -----------                            -------------------
Engineering Attributes for cloud engineering team  25
HR          Attributes for HR team                 15
Hackathon   Hackathon attribute set                20
```

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraAttributeSet.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+```powershell
+Connect-Entra -Scopes 'CustomSecAttributeDefinition.Read.All'
+Get-EntraAttributeSet -Id 'Engineering'
+```
+
+```Output
+Id          Description                           MaxAttributesPerSet
+--          -----------                           -------------------
+engineering Attributes for cloud engineering team 25
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraAttributeSet.md:82]:
```suggestion
Engineering  Attributes for cloud engineering team  25
```

**Review by SteveMutungi254 — APPROVED** [review]
LGTM.

**Pull request merged**